prodtests/full-system-test/aggregator-workflow.sh

```diff
@@ -44,67 +44,120 @@ if [[ $BEAMTYPE == "PbPb" ]]; then
 fi
 
 # Calibration workflows
-if ! workflow_has_parameter CALIB_LOCAL_INTEGRATED_AGGREGATOR; then WORKFLOW=; fi
+if ! workflow_has_parameter CALIB_LOCAL_INTEGRATED_AGGREGATOR; then
+    WORKFLOW=
+else
+    if [[ -z $AGGREGATOR_TASKS ]]; then
+  ech HERE1
+    AGGREGATOR_TASKS=ALL
+  fi
+fi
+
+# Which calibrations are we aggregating
+echo "AGGREGATOR_TASKS = $AGGREGATOR_TASKS" 1>&2
 
 # adding input proxies
 if workflow_has_parameter CALIB_PROXIES; then
-  if [[ ! -z $CALIBDATASPEC_BARREL ]]; then
-    add_W o2-dpl-raw-proxy "--dataspec \"$CALIBDATASPEC_BARREL\" $(get_proxy_connection barrel input)" "" 0
-  fi
-  if [[ ! -z $CALIBDATASPEC_CALO ]]; then
-    add_W o2-dpl-raw-proxy "--dataspec \"$CALIBDATASPEC_CALO\" $(get_proxy_connection calo input)" "" 0
+  if [[ $AGGREGATOR_TASKS == BARREL_TF ]]; then
+      if [[ ! -z $CALIBDATASPEC_BARREL_TF ]]; then
+    add_W o2-dpl-raw-proxy "--dataspec \"$CALIBDATASPEC_BARREL_TF\" $(get_proxy_connection barrel_tf input)" "" 0
+      fi
+  elif [[ $AGGREGATOR_TASKS == BARREL_SPORADIC ]]; then
+      if [[ ! -z $CALIBDATASPEC_BARREL_SPORADIC ]]; then
+    add_W o2-dpl-raw-proxy "--dataspec \"$CALIBDATASPEC_BARREL_SPORADIC\" $(get_proxy_connection barrel_sp input)" "" 0
+      fi
+  elif [[ $AGGREGATOR_TASKS == CALO_TF ]]; then
```

Comment:
Also the indentation seems strange. Why isn't the `add_W ...` indented?

Reply:
This is the result from clang-format + `sed -i 's/[[:space:]]*$//'` and `sed -i 's/\t/  /g'`
I can check again.

Reply:
I will check.

Reply:
Emacs was putting 4 spaces - no idea why, it always worked. Now I changed to 2, let's see if it works...